### PR TITLE
[Post] console config

### PR DESCRIPTION
**Pull request by TomasVotruba**
Ref #320

### CODE REVIEW

File: source/_posts/2017/2017-06-01-thank-you-david.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 id: 40
-title: "Thank you David"
+title: "Thank you David Grudl"
```
Comment (Lexinek) [inline source/_posts/2017/2017-06-01-thank-you-david.md]: I consider my own (similar) post as huge step in my life.  @TomasVotruba thank you for reminder!

Comment (Lexinek) [reply source/_posts/2017/2017-06-01-thank-you-david.md]: Anyways, would you mind renaming the file, adding surname `grudl` in it?

Comment (TomasVotruba) [reply source/_posts/2017/2017-06-01-thank-you-david.md]: Thanks for last kick of courage that helped me to do it :)

Any open-soucing plans with that post? :P

**Comment by Lexinek** [comment]
redireeect old -> new 😄

**Comment by TomasVotruba** [comment]
Prirorities :)

**Pull request merged**